src/components/Home/Home.jsx

```diff
@@ -1,8 +1,48 @@
 import PropTypes from 'prop-types';
+import { useState, useEffect } from 'react';
+import { useApolloClient, gql } from '@apollo/client';
 import './Home.css';
 import EncounterPreviewContainer from '../EncounterPreviewContainer/EncounterPreviewContainer';
 
-export default function Home({ encounters }) {
+export default function Home({ userName }) {
+  const [encounters, setEncounters] = useState([]);
+  const client = useApolloClient();
+
+  useEffect(() => {
+    const getEncounters = async (userName) => {
+      if (userName) {
+        try {
+          const { data } = await client.query({
```

Comment:
This is so nice to see. It definitely is a little messier consuming a graphql API on the rails side of things.